src/config.rs

```diff
@@ -195,6 +200,26 @@ pub fn construct_config(pc: ProxyConfig) -> Result<Config, Error> {
         Some(parse_default(CA_ADDRESS, default_istiod_address)?)
     });
 
+    let xds_root_cert_provider = parse_default(
+        XDS_ROOT_CERT_PROVIDER_ENV,
+        DEFAULT_ROOT_CERT_PROVIDER.to_string(),
+    )?;
+    let xds_root_cert = if Path::new(&xds_root_cert_provider).exists() {
```

Comment:
can we handle SYSTEM special case as well?

Reply:
Okay, will fix it later